### DISCUSSION TITLE
Would love to have an option to export 2FA codes as QR code

### DISCUSSION THREAD

**Discussion by Srijith-Seetharaman** [ente-io/ente#3774] [Enhancements]
Right now, users are only able to export their 2FA as text, which is not very useful, especially if they want to port them into other Authenticator apps like Google Authenticator or Microsoft Authenticator. So, it would be very useful if you could serve up a QR code that can be scanned and immediately exported into other authenticator apps. What say you, community?

**Comment by ua741** [comment]
Related to https://github.com/ente-io/ente/discussions/500
